### ISSUE TITLE
[BUG] can't run example zinc_protein

### ISSUE THREAD

**Issue by xusuyong** [deepmodeling/deepmd-kit#3061]
### Bug summary

can't run example zinc_protein

### DeePMD-kit Version

2.2.7

### TensorFlow Version

2.10

### How did you download the software?

pip

### Input Files, Running Commands, Error Log, etc.

dp train .\zinc_se_a_mask.json

### Steps to Reproduce

PS D:\xsy\PycharmProjects\deepmd-kit\examples\zinc_protein> dp train .\zinc_se_a_mask.json
WARNING:tensorflow:From C:\Users\DELL\miniconda3\envs\tf2\lib\site-packages\tensorflow\python\compat\v2_compat.py:107: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
Instructions for updating:
non-resource variables are not supported in the long term
WARNING:root:To get the best performance, it is recommended to adjust the number of threads by setting the environment variables OMP_NUM_THREADS, TF_INTRA_OP_PARALLELISM_THREADS, and TF_INTER_OP_PARALLELISM_THREADS. See https://deepmd.rtfd.io/parallelism/ for more information.
WARNING:root:Environment variable KMP_BLOCKTIME is empty. Use the default value 0
WARNING:root:Environment variable KMP_AFFINITY is empty. Use the default value granularity=fine,verbose,compact,1,0
C:\Users\DELL\miniconda3\envs\tf2\lib\importlib\__init__.py:169: UserWarning: The NumPy module was reloaded (imported a second time). This can in some cases result in small but subtle issues and is discouraged.
  _bootstrap._exec(spec, module)
WARNING:deepmd.train.run_options:Switch to serial execution due to lack of horovod module.
DEEPMD INFO    Calculate neighbor statistics... (add --skip-neighbor-stat to skip this step)
Traceback (most recent call last):
  File "C:\Users\DELL\miniconda3\envs\tf2\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\DELL\miniconda3\envs\tf2\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\DELL\miniconda3\envs\tf2\Scripts\dp.exe\__main__.py", line 7, in <module>
  File "D:\xsy\PycharmProjects\deepmd-kit\deepmd_cli\main.py", line 635, in main
    deepmd_main(args)
  File "D:\xsy\PycharmProjects\deepmd-kit\deepmd\entrypoints\main.py", line 74, in main
    train_dp(**dict_args)
  File "D:\xsy\PycharmProjects\deepmd-kit\deepmd\entrypoints\train.py", line 149, in train
    jdata = update_sel(jdata)
  File "D:\xsy\PycharmProjects\deepmd-kit\deepmd\entrypoints\train.py", line 514, in update_sel
    jdata_cpy["model"] = Model.update_sel(jdata, jdata["model"])
  File "D:\xsy\PycharmProjects\deepmd-kit\deepmd\model\model.py", line 507, in update_sel
    return cls.update_sel(global_jdata, local_jdata)
  File "D:\xsy\PycharmProjects\deepmd-kit\deepmd\model\model.py", line 664, in update_sel
    local_jdata_cpy["descriptor"] = Descriptor.update_sel(
  File "D:\xsy\PycharmProjects\deepmd-kit\deepmd\descriptor\descriptor.py", line 511, in update_sel
    return cls.update_sel(global_jdata, local_jdata)
  File "D:\xsy\PycharmProjects\deepmd-kit\deepmd\descriptor\se.py", line 162, in update_sel
    return update_one_sel(global_jdata, local_jdata_cpy, False)
  File "D:\xsy\PycharmProjects\deepmd-kit\deepmd\entrypoints\train.py", line 480, in update_one_sel
    rcut = descriptor["rcut"]
KeyError: 'rcut'

**Comment by njzjz** [comment]
cc: @dingye18

**Comment by dingye18** [comment]
Since the `se_a_mask` descriptor could accept the dummy atoms as input in coords, the neighbors stat calculation is meaningless for `se_a_mask`. 
Thus, for example training of zinc_protein.json, `--skip-neighbor-stat` flag is needed.
Try run this script from the source dir of deepmd-kit.
```shell
dp train --skip-neighbor-stat ./examples/zinc_protein/zinc_se_a_mask.json
```` 
BTW, I just found that the path to examples data directory is wrong in zinc_se_a_mask.json. 
It was fixed in PR #3068
